internal/functions/efgs/upload-batch.go

```diff
@@ -0,0 +1,159 @@
+package efgs
+
+import (
+	"bytes"
+	"context"
+	b64 "encoding/base64"
+	"encoding/json"
+	"errors"
+	efgsutils "github.com/covid19cz/erouska-backend/internal/functions/efgs/utils"
+	"github.com/covid19cz/erouska-backend/internal/logging"
+	"github.com/covid19cz/erouska-backend/internal/secrets"
+	"github.com/golang/protobuf/proto"
+	"io/ioutil"
+	"log"
+	"net/http"
+	urlutils "net/url"
+	"time"
+)
+
+type uploadResponse struct {
+	Error     []int `json:"500"`
+	Duplicate []int `json:"409"`
+	Success   []int `json:"201"`
+}
+
+//UploadBatch Called in CRON. Get keys from database and upload them to EFGS.
+func UploadBatch() {
+	ctx := context.Background()
+	logger := logging.FromContext(ctx)
+	currentTime := time.Now()
```

Comment:
`now`? :thinking:

Reply:
Any comment?

Reply:
yes, now. I want to select keys from last 14 days.